BuyBitcoin/settings.py

```diff
@@ -92,7 +95,12 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
-DATABASES = {}
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
```

Comment:
I don't think this doese anything

Reply:
It does, switches local backend to sqlite, which means that you can easily create/destroy data. its the heroku recommended fashion of doing dbs (sqlite in dev, postgresql in prod).